tests/cluster/parameterize_utils.py

```diff
@@ -406,6 +406,10 @@ def create_deployment(deployment_ident, image, args, environ, ports, replicas):
                     "containers": [container, sidecar_container],
                 },
             },
+            "hostAliases": [{
+                "ip": "127.0.0.1",
+                "hostnames": ["foo.local"]
+            }]
```

Comment:
The `hostAliases` field needs to be in the Pod spec, not the Deployment spec. This Deployment does not validate.

Reply:
moved